packages/rockets-next-web/src/components/CrudModule/README.md

```diff
@@ -0,0 +1,130 @@
+# CrudModule
+
+## **TableProps**
+
+Props that modify the layout and/or functionality of the module Table.
+
+### **tableSchema**
+
+Array of union types containing `HeaderProps` (imported from `@concepta/react-material-ui/dist/components/Table/types`) and additional atributes representing columns that should be rendered by the table. Each arary item can contain `id`, `label`, `disablePadding`, `width`, `numeric`, `textAlign`, `sortable` and `format` attributes, being `id` and `label` the only required ones.
+
+Important to mention that `format` represents a custom format for the column data.
+
+```js
+[
+  {
+    id: "fullName", // required
+    label: "Full Name", // required
+    disablePadding: false,
+    width: 100,
+    numeric: false,
+    textAlign: "left" | "center" | "right",
+    sortable: true,
+    format: (value: string | number) => new Date(value).toString(),
+  },
+];
+```
+
+**type**: `HeaderProps[]`\
+**default**: `ID`, `Email`, `Username` and `Actions`
+**required**: `false`
+
+### **searchParam**
+
+ID of a column data by which table items can be filtered when changing the search input value.
+
+**type**: `string`\
+**default**: `email`\
+**required**: `false`
+
+### **hideActionColumn**
+
+Actions table column is visible by default. This prop is responsible for changing that.
+
+**type**: `boolean`\
+**default**: `false`\
+**required**: `false`
+
+### **overrideDefaults**
```

Comment:
This prop is showing twice, but according to its description, they seem to be different.
Maybe it's just a mistake in the prop name